httpstan/models.py

```diff
@@ -229,7 +275,14 @@ def _build_extension_module(
         # TODO(AR): wrap stderr, return it as well
         build_extension.run()
 
-        module = _load_module(module_name, build_extension.build_lib)
-        with open(module.__file__, "rb") as fh:  # type: ignore  # pending fix, see mypy#3062
+        if sys.platform.startswith("win"):
+            # tempfile bug: Windows can't open file that is already open
+            # see related statement in https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile
+            path = _find_module(module_name, build_extension.build_lib)
+            with open(path, "rb") as fh:  # type: ignore  # pending fix, see mypy#3062
+                return fh.read()  # type: ignore  # pending fix, see mypy#3062
+        else:
+            module = _load_module(module_name, build_extension.build_lib)
```

Comment:
see comment above. I think _find_module (or maybe renamed, _find_module_path) might replace the old _load_module

Reply:
Is it ok to remove the assert (it tests againts loaded module)

     assert module.__name__ == module_name, (module.__name__, module_name)